site/tests/app/models/ButtonTester.php

```diff
@@ -47,12 +47,12 @@ public function testOverrides() {
     }
 
     public function testWrongTypeProgress() {
-        $button = new Button($this->createMockCore(), ['progress' => 'a']);
+        $button = new Button(new Core(), ['progress' => 'a']);
         $this->assertEquals(0, $button->getProgress());
     }
 
     public function testDate() {
-        $button = new Button($this->createMockCore(), []);
+        $button = new Button(new Core(), []);
```

Comment:
These changes are to remove the usage of the mock which is no longer necessary due to prior work on making the Core constructor have less initialization in it. This is mostly a slight optimization to speed-up running the test suite as it's quicker to make a `Core` object than it is to make a mock of it. This applies elsewhere in this file.